lib/internal/event_target.js

```diff
@@ -973,6 +834,98 @@ ObjectDefineProperties(NodeEventTarget.prototype, {
 
 // EventTarget API
 
+/**
+ * @see https://dom.spec.whatwg.org/#dictdef-eventlisteneroptions
+ * @typedef {Object} AddEventListenerOptions
+ * @property {null} __proto__
+ * @property {boolean} [capture=false]
+ * @property {boolean} passive
+ * @property {boolean} [once=false]
+ * @property {import('./abort_controller').AbortSignal|null} signal
+ */
+
+/**
+ * @see https://dom.spec.whatwg.org/#event-flatten-more
+ * @param {*} options
+ * @returns {AddEventListenerOptions}
+ */
+function flattenMoreOptions (options) {
+  if (typeof options === 'boolean') {
```

Comment:
I recommend benchmarking but, `options === true || options === false` might be faster.